game/src/MainMenu/MainMenu.gd

```diff
@@ -12,23 +12,28 @@ func _ready():
 
 
 func _on_new_game_button_pressed():
+	await get_node("/root/GameMenu/SFX").play("click")
 	print("Start a new game!")
 	get_tree().change_scene_to_file("res://src/SampleGame.tscn")
 
 
 func _on_continue_button_pressed():
+	get_node("/root/GameMenu/SFX").play("click")
```

Comment:
This is going to be a problem eventually. You want a more uniform, and less error prone, way of making the click sound play for all button presses without having to manually add a command to each button action.

Reply:
No it's not

Reply:
I completely agree, but that's not my purpose with this PR; I'm adding a way to play sound effects, not a way to automagically call it when buttons are pressed.

Reply:
Since this is a common thing (add sounds to button clicks) there is probably some way built into Godot to handle it without manually adding the sound in the function body